common/rdr/RandomStream.cxx

```diff
@@ -69,7 +69,7 @@ RandomStream::RandomStream()
   {
 #endif
 #endif
-    vlog.error("no OS supplied random source - using rand()");
+    vlog.error("No OS supplied random source: Using rand()");
```

Comment:
A comma suggests an explanation or more details. Perhaps a comma if the dash is undesirable?

Reply:
A stray `r` sneaked in here instead. Otherwise, it looks good. :)

Reply:
Nice catch, fixed now :)